web_driver.sh

```diff
@@ -57,6 +58,26 @@ function runUITests ()
   runInTestDirAndAssertCmd "python ui_test_suite.py --server_url='$SERVER_URL' --username='$USERNAME' --password='$PASSWORD'"
 }
 
+function runRemoteUITests ()
```

Comment:
nit: I am not sure what is meant by "Remote" in the function name?  Would this rather be a local server in the travis environment?  If so, maybe `runFunctionalUITests()` might be better naming?